msi/nunit/engine-files.wxi

```diff
@@ -32,11 +32,11 @@
                     <CopyFile Id="nunit.agent.net40.engine.api.xml" DestinationDirectory="NET40_AGENT_DIR" />
                 </File>
             </Component>
-            <Component Id="MONO.CECIL" Location="local" Guid="D4A15752-6685-3494-9346-F623175CEFFA">
-                <File Id="Mono.Cecil.dll"
-                      Source="$(var.InstallImage)bin/net20/Mono.Cecil.dll">
-                    <CopyFile Id="nunit.agent.net20.Mono.Cecil.dll" DestinationDirectory="NET20_AGENT_DIR" />
-                    <CopyFile Id="nunit.agent.net40.Mono.Cecil.dll" DestinationDirectory="NET40_AGENT_DIR" />
+            <Component Id="NUNIT.ENGINE.METADATA" Location="local" Guid="D4A15752-6685-3494-9346-F623175CEFFA">
```

Comment:
Shouldn't we be changing the GUID here, as this is a new and different assembly? I think that would be best practice.

Reply:
I'm not sure I know the right answer. The GUID represents the component, not the assembly. I figured that it's essentially a replacement component and so should have the same GUID. I believe that would be necessary if we did partial updates, e.g. replacing just the one component.

Since we always do a full install, I believe either will work, but I'm not 100% sure which approach is technically more correct. I sent Rob Mensching a note to see what he thinks.

Reply:
Got an answer... Per Rob, I should change the component GUID if the name of the assembly it contains is changing. I'll go ahead and do that.

FWIW, he also suggests we stop maintaining GUIDs on components and let WIX take care of it for us. That seems like a separate issue, however.

Reply:
I didn’t know that was an option - I’ve always done my own GUIDs. Cool!

Reply:
Done